frontend/survey-project/src/app/components/pages/profile-page/ui/general-informations/general-informations.component.ts

```diff
@@ -0,0 +1,119 @@
+import { ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { UserService } from '../../../../../shared/services/user.service';
+import { ToastService } from '../../../../../shared/services/toast.service';
+import { FormBuilder, FormGroup } from '@angular/forms';
+import { GeneralInfoResponse } from '../../../../../shared/dto/responses/user/general-info-response';
+import { ChangeGeneralInformationRequest } from '../../../../../shared/dto/requests/user/change-general-info-request';
+import { ChangeGeneralInformationResponse } from '../../../../../shared/dto/responses/user/change-general-info-response';
+import { CommonDialogsService } from '../../../../../shared/services/commondialog.service';
+
+@Component({
+  selector: 'app-general-informations',
+  templateUrl: './general-informations.component.html',
+  styleUrl: './general-informations.component.scss'
+})
+export class GeneralInformationsComponent implements OnInit {
+  userInfo: any = null;
+  generalInfoForm!: FormGroup;
+
+  constructor(
+    private readonly userService: UserService,
+    private readonly toastService: ToastService,
+    private readonly fb: FormBuilder,
+    private readonly cdr: ChangeDetectorRef,
+    private readonly commonDialogs: CommonDialogsService
+  ) {
+    this.generalInfoForm = this.fb.group({
```

Comment:
You didn't add any validatiors for this FormGroup?
Maybe required so that error handling is easier.